src/features/profile/components/collection-container/scores/ingredients/index.tsx

```diff
@@ -0,0 +1,215 @@
+/* eslint-disable consistent-return */
+/* eslint-disable array-callback-return */
+import React, { useContext, useState, useEffect } from 'react';
+import { Image, Typography, Space, Card, Rate, Divider } from 'antd';
+import { IngredientsRepositoryContext } from 'context/ingredients';
+import { DefaultGrade, DefaultIngredient } from 'context/ingredients/constants';
+import { DefaultScore, DefaultSummer2021Score } from 'context/scores/constants';
+import { ScoresRepositoryContext } from 'context/scores';
+import { AuthenticationRepositoryContext } from 'context/authentication';
+import { DefaultDancer, DancersRepositoryContext } from 'context/dancer';
+import { SongsRepositoryContext } from 'context/songs';
+import { DefaultSong } from 'context/songs/constants';
+import { JacketDifficulty, IngredientsContainer } from './styled';
+
+const { Text, Paragraph, Title } = Typography;
+
+const Ingredients = () => {
```

Comment:
Since this is part of the profile, it should bring the dancer object in from the parent component (ie. the dancer). This is so if you visit someone else's profile, you will see their ingredients and not your own.